modules/s3cfg.py

```diff
@@ -3054,6 +3054,12 @@ def get_req_requester_from_site(self):
         """
         return self.req.get("requester_from_site", False)
 
+    def get_req_task_list_field(self):
+        """
+            Whether to unable to the task field in the list view
```

Comment:
Typo: unable - enable.  Or better, "show".  Also, can you say which list view this is for?